src/Google/Ads/GoogleAds/Lib/V7/GoogleAdsClientBuilder.php

```diff
@@ -80,6 +88,20 @@ public function from(Configuration $configuration)
         );
         $this->proxy = $configuration->getConfiguration('proxy', 'CONNECTION');
         $this->transport = $configuration->getConfiguration('transport', 'CONNECTION');
+        $this->grpcChannelIsSecure =
+            is_null($configuration->getConfiguration('grpcChannelIsSecure', 'CONNECTION'))
+            || $configuration->getConfiguration('grpcChannelIsSecure', 'CONNECTION') === ""
```

Comment:
Can you double check if `getConfiguration` can return null as well? If yes, we probably need to use `empty` instead.

Reply:
I cannot use `empty` here because `empty(false)` returns `true`. We only want to use the default value when the value is undefined or an empty string.

Reply:
Sorry. Somehow, I didn't see line 92 that you checked is_null already.